src/icons.qrc

```diff
@@ -354,7 +354,7 @@
         <file>icons/skin/paused.png</file>
         <file>icons/skin/qbittorrent-tray-dark.svg</file>
         <file>icons/skin/qbittorrent-tray-light.svg</file>
-        <file>icons/skin/qbittorrent16.png</file>
+        <file alias="www/favicon.ico">icons/skin/qbittorrent16.png</file>
```

Comment:
It is possible to move `www/favicon.ico` to cpp code? it's quite hard to find here.

Reply:
or move it to HTML?

Reply:
>It is possible to move www/favicon.ico to cpp code?

It was previous behavior.
>or move it to HTML?

What do you mean?

Reply:
>What do you mean?

https://en.wikipedia.org/wiki/Favicon#How_to_use

Reply:
@Chocobo1: how a creator of a alternative UI would get the true icon name?

Reply:
>@Chocobo1: how a creator of an alternative UI would get the true icon name?

Now I think we should properly introduce an ICO format favicon instead of creating an alias of the png.
We can do it later if you won't handle it now.

Reply:
>how a creator of an alternative UI would get the true icon name?

He just add the icon he want into his www folder...

Reply:
@Chocobo1, `<link rel="icon" type="image/png" href="images/skin/qbittorrent16.png">` is used now.

Reply:
OK thank you!